windows/activities.md

```diff
@@ -94,6 +95,7 @@ The commands for the various functions are as follows:
 * "***Who are you/Identify Youself/What is your name***" for B.E.N.J.I. to introduce itself.
 * Face detection feature will automatically fetch image for first time user. From next time, it will predict the authenticity of user. If user is authenticated, then B.E.N.J.I. will start running. If user wants to add another user, he/she can use "***face*** username". By default, username is "admin". So, make sure, while adding a new user, use "username" other than "admin". B.E.N.J.I. will fetch a single image of the user and store it in "dataset" folder in "C" drive. 
 NOTE: Make sure that system is able to fetch a good image of yours(be in a good lighting conditions). You can also directly add image to "dataset" folder in "C" drive.
+* "***lyrics*** _SongName_" gives lyrics of the mentioned song. If required, more details can be provided after the _SongName_.
```

Comment:
Can you please specify the format of "more" details. eg: `abc movie:123` or `abc 123` or `abc album 123`